FSharpLu.Json/Compact.fs

```diff
@@ -10,18 +13,33 @@ open Microsoft.FSharp.Reflection
 type CompactUnionJsonConverter() =
     inherit Newtonsoft.Json.JsonConverter()
 
+    let processWithContractResolver (serializer:JsonSerializer) (name:string) =
```

Comment:
Hmm... We can walk around the dynamic match by
1) creating two separate Converters - one for PascalCase, one for camelCase
2) creating two separate ContractResolvers (with function as above) - one would derive from Default, the other from CamelCase. See http://www.newtonsoft.com/json/help/html/Performance.htm  snippet "Use JsonConverter with IContractResolver".